src/controllers/admin.js

```diff
@@ -0,0 +1,31 @@
+import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
+import dbClient from '../utils/dbClient.js'
+
+export const updateUserRoleById = async (req, res) => {
+  const id = Number(req.params.id)
+  const { content } = req.body
+  const foundUserRole = req.user.role
+
+  const foundUser = await dbClient.user.findUnique({
+    where: { id, content }
+  })
+
+  if (!foundUser) {
+    return sendDataResponse(res, 404, 'User not found')
+  }
+  if (foundUserRole === 'ADMIN') {
```

Comment:
The only time this controller will respond back to the client is within this if statement. This means that if, for whatever reason, this if statement doesn't run then the request will hang and eventually timeout.

Reply:
Should I remove it?

Reply:
Changed sendDataResponse to sendMessageResponse

Reply:
You'll still need to add an else statement to your if statement to ensure that this function will always return a response to the user.

Reply:
added an else statement with res code 403